jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/CalendarsTests.java

```diff
@@ -171,54 +159,52 @@ public void numberOfIntervalsTest() {
 
         int expectedNumberOfIntervals = 24;
         typeDayCalendar.has()
-                       .numberOfIntervals(expectedNumberOfIntervals);
+                .numberOfIntervals(expectedNumberOfIntervals);
     }
 
-    @Test(
-        description = "Check dark theme",
-        enabled = false
+    @Test(description = "Check dark theme",
+            enabled = false
     ) // fix theme on the test site
     public void darkCalendarTest() {
         darkCalendar.show();
         darkCalendar.has().darkTheme();
     }
 
-    @Test(description = "Check setting date through a text input field")
+    @Test(description = "Check setting date through a text input field",
+            enabled = false)
```

Comment:
Why do you disable test? :)

Reply:
Because separating calendar into to objects, this input field is "lost" and can not be a part of a calendar or a toolbar.
And in the last version of Vuetify there is no such field within a calendar. I disabled it temporarily, we need to decide what to do with it.